### DISCUSSION TITLE
SD WebUI Forge

### DISCUSSION THREAD

**Discussion by FilipeF12** [LykosAI/StabilityMatrix#1016] [Q&A]
Where do I check which version of WebUI Forge is installed in Stability Matrix? On the Packages tab SD WebUI lists the version (v1.10.0), but under WebUI Forge it says (main@c9550c8)??

**Comment by mohnjiles** [comment]
It should show it in the console during startup:
![image](https://github.com/user-attachments/assets/caab2a4f-0212-4dba-85d4-fa9d039c1708)


or at the bottom of the web UI:
![image](https://github.com/user-attachments/assets/56372ec9-e15f-4ccf-9034-f1375cd87625)

**Comment by FilipeF12** [comment]
So package 'Stable Diffusion WebUI' shows the version where the packages are listed, pre-launch, and 'Stable Diffussion WebUI Forge' must be launched to see the version. Got it.
